lib/teleterm/daemon/gateway_cert_reissuer.go

```diff
@@ -129,18 +131,30 @@ func (r *GatewayCertReissuer) reissueCert(ctx context.Context, gateway *gateway.
 		return trace.Wrap(err)
 	}
 
-	err = r.reissueAndReloadGatewayCert(ctx, gateway, dbCertReissuer)
+	err = r.reissueAndReloadGatewayCert(ctx, gateway, certReissuer)
 	if err != nil {
 		return trace.Wrap(err)
 	}
 
 	return nil
 }
 
-func (r *GatewayCertReissuer) reissueAndReloadGatewayCert(ctx context.Context, gateway *gateway.Gateway, dbCertReissuer DBCertReissuer) error {
-	err := dbCertReissuer.ReissueDBCerts(ctx, gateway.RouteToDatabase())
-	if err != nil {
-		return trace.Wrap(err)
+func (r *GatewayCertReissuer) reissueAndReloadGatewayCert(ctx context.Context, gateway *gateway.Gateway, certReissuer CertReissuer) error {
+	switch {
+	case uri.IsDB(gateway.TargetURI()):
+		err := certReissuer.ReissueDBCerts(ctx, gateway.RouteToDatabase())
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+	case uri.IsKube(gateway.TargetURI()):
+		err := certReissuer.ReissueKubeCerts(ctx, gateway.TargetName())
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+	default:
+		return trace.NotImplemented("gateway not supported for %v", gateway.URI())
```

Comment:
I'd mention reloading certs in the error message so that it can be differentiated from the error message returned from `clusters.Cluster.CreateGateway`.